Tracker/Controllers/NewTrackerViewController.swift

```diff
@@ -0,0 +1,258 @@
+        button.setTitle("Создать", for: .normal)
+        button.setTitleColor(.trWhite, for: .normal)
+        button.titleLabel?.font = UIFont.systemFont(ofSize: 16, weight: .medium)
+        button.backgroundColor = .trGray
+        button.layer.cornerRadius = 16
+        button.clipsToBounds = true
+        button.addTarget(self, action: #selector(createButtonTapped), for: .touchUpInside)
+        button.isEnabled = false
+        return button
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        view.backgroundColor = .trWhite
+        addSubviews()
+        addConstraints()
+    }
+    
+    func addSubviews() {
+        [titleLabel,textField, tableView, saveButton, cancelButton].forEach {
+            $0.translatesAutoresizingMaskIntoConstraints = false
+            view.addSubview($0)
+        }
+    }
+    
+    func addConstraints() {
+        switch typeOfNewTracker {
+        case .habitTraker: heightTableView = 149
+        case .eventTraker: heightTableView = 74
+        case .none: break
+        }
+        
+        NSLayoutConstraint.activate([
+            titleLabel.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            titleLabel.bottomAnchor.constraint(equalTo: view.topAnchor, constant: 40),
+            
+            textField.topAnchor.constraint(equalTo: titleLabel.bottomAnchor, constant: 24),
+            textField.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 16),
+            textField.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -16),
+            textField.heightAnchor.constraint(equalToConstant: 75),
+            textField.widthAnchor.constraint(equalTo: view.widthAnchor, constant: -32),
+            
+            tableView.topAnchor.constraint(equalTo: textField.bottomAnchor, constant: 24),
+            tableView.leadingAnchor.constraint(equalTo: textField.leadingAnchor),
+            tableView.trailingAnchor.constraint(equalTo: textField.trailingAnchor),
+            tableView.heightAnchor.constraint(equalToConstant: heightTableView),
+            
+            cancelButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 16),
+            cancelButton.trailingAnchor.constraint(equalTo: saveButton.leadingAnchor, constant: -8),
+            cancelButton.heightAnchor.constraint(equalToConstant: 50),
+            cancelButton.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor),
+            
+            saveButton.heightAnchor.constraint(equalTo: cancelButton.heightAnchor),
+            saveButton.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -16),
+            saveButton.centerYAnchor.constraint(equalTo: cancelButton.centerYAnchor),
+            saveButton.widthAnchor.constraint(equalTo: cancelButton.widthAnchor)
+        ])
+    }
+    
+    @objc private func cancelButtonTapped() {
+        dismiss(animated: true) {
+        }
```

Comment:
Попробуй убрать, кажется излишне)
```suggestion
        dismiss(animated: true)
```